content/blog/2019/09/2019-09-16-outreachy-audit-log-release.adoc

```diff
@@ -0,0 +1,40 @@
+By default, audit logs will be written in HTML files (rotated once per day) to `$JENKINS_HOME/logs/html/audit.html` which are viewable through the "Audit Logs" root action link.
+In the system settings, a section for audit logging is added where the main audit log output can be configured.
+This can initially be configured to output via either a JSON log file in `$JENKINS_HOME/logs/audit.log` by default or to a syslog server using RFC5424 encoding.
+
+Overall, this experience has been rather interesting.
+Besides having an opportunity to mentor new contributors, Outreachy has helped open my eyes to the struggles that developers from around the world are dealing with which can be improved upon to help expand our communities.
+For example, many countries do not have reliable internet or electricity, so the use of synchronous videoconferencing and other heavyweight, synchronous processes common to more corporate-style development are inadequate in this international context.
+This doesn't even begin to account for the difference in timezones which is not always an issue, though both problems are addressable by using asynchronous communication methods like chat and email.
+This notion of asynchronous communication is an important aspect of the Apache Way, for example, which emphasises processes that allow for vendor neutral communities to form and thrive around a project.
+
+This mentoring project was valuable to myself as well.
+This audit log project was not led by any project managers, so this gave me an opportunity to develop my own skills in that area since I'm normally programming myself.
+The drastic change of feature discovery from experimenting directly with what might be possible to discussions around having other people implement them was certainly something difficult to get used to.
+In order to appropriately scope the project, I still had to spend a bit of time reading through the Jenkins codebase to determine which tasks could be implemented simply (e.g., good newbie-friendly issues), which tasks might require changes to Jenkins itself (previously discovered to take too long for these relatively short Outreachy rounds), and which tasks would require intimate familiarity with Jenkins and would likely be infeasible for new developers to Jenkins.
+
+Overall, I think we all did a good job of managing scope of this project without spending an inordinate amount of time on any single feature without knowledge of users' desires.
+The first release of this plugin is now available in the Jenkins Update Center.
+In the future, I hope to learn more about developing Jenkins UI components so that we can create a more dynamic and Jenkins-like configuration page for choosing where logs are output.
+While I don't intend on using this plugin for further Outreachy rounds, I do hope to see more interest in it over time as the more security-conscious users out there discover this new plugin.
```

Comment:
Only other suggestion/optimization is to add a screenshot and mark appropriately so when we tweet out a link the Twitter card looks good.

Reply:
Done in e4999df4d5afcf85ac719be8f407ebb6ce4df690